src/xtgeo/grid3d/_gridprop_op1.py

```diff
@@ -135,7 +148,7 @@ def operation_polygons(self, poly, value, opname="add", inside=True):
     elif opname == "div":
         # Dividing a map of zero is always a hazzle; try to obtain 0.0
         # as result in these cases
-        if 0.0 in value:
+        if value == 0.0:
```

Comment:
This looks flaky, consider using np.isclsoe (https://numpy.org/doc/stable/reference/generated/numpy.isclose.html#numpy-isclose)

```suggestion
        if np.isclose(value, 0):
```

Reply:
Done 👍